pkgs/tools/system/smartmontools/default.nix

```diff
@@ -2,13 +2,13 @@
 , IOKit ? null , ApplicationServices ? null }:
 
 let
-  version = "7.0";
+  version = "7.1";
 
-  dbrev = "4883";
-  drivedbBranch = "RELEASE_${builtins.replaceStrings ["."] ["_"] version}_DRIVEDB";
+  dbrev = "5033";
+  drivedbBranch = "RELEASE_7_0_DRIVEDB";
```

Comment:
Note: there is no dedicated branch for 7.1. The 7.0 branch is supposed to be used for both 7.0 and 7.1.